<ui version="4.0" >
 <class>AdvancedDialog</class>
 <widget class="QWidget" name="AdvancedDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>advancedDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>&amp;Interface</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <item>
         <widget class="QCheckBox" name="m_useMillisecondsCheckBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Use &amp;milliseconds instead of seconds</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title" >
        <string>&amp;Controls</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QCheckBox" name="m_enableMouseWheelCheckBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip" >
           <string>Enable mouse wheel to move between slides.</string>
          </property>
          <property name="text" >
           <string>Enable mouse &amp;wheel </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>&amp;Ken Burns effect</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QCheckBox" name="m_kbDisableFadeCheckBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Disable &amp;fade-in / fade-out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_kbDisableCrossfadeCheckBox" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Disable &amp;cross-fade</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>&amp;Others</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <widget class="QCheckBox" name="m_cacheCheckBox" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text" >
           <string>Enable &amp;Cache</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="m_cacheSizeLabel1" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>Cache size:</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="m_cacheSizeSpinBox" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="MinimumExpanding" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimum" >
             <number>5</number>
            </property>
            <property name="maximum" >
             <number>20</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_cacheSizeLabel2" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Minimum" hsizetype="Minimum" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text" >
             <string>images</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="m_KBCacheLabel" >
        <property name="enabled" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string>&lt;b>Notice&lt;/b>:
Ken Burns effect does not use this cache mechanism.</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>142</width>
       <height>119</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
